extensions/exposition/components/octets.storage/manifest.toa.yaml

```diff
@@ -10,6 +10,7 @@ operations:
       storage*: string
       request*: ~
       accept: string
+      limit: number
```

Comment:
I'm confused - the documentation says it can be either integer or a string?

Reply:
Documentation describes Exposition human-declaration format which is parsed on startup. 

This is an application interface of another extension (Storages), which doesn't know anything about human declarations in the Exposition, and only accepts numbers.

Reply:
I assume it's a number of bytes then? Why it's not the `integer` in that case?

Reply:
That's the limit after which the amount of streaming bytes will bee considered exceeding.
Only `integers` make sense here, but technically it can be a floating point number.

Switching this to integer will require much more complicated parsing of human declaration (`1.5MB` is valid while `1.5B` is not). There is no reason to implement it.